### PR TITLE
Change health via difficulty

### PR DESCRIPTION
**Pull request by DabbinDude316B**

### CODE REVIEW

**Comment by Itz-Miles** [comment]
You might want to merge this functionality with the gameplay changers, resulting in gameplay settings per difficulty.

**Review by rodney528 — APPROVED** [review]
Meh

File: source/Note.hx [hunk 1]
```diff
@@ -265,6 +265,18 @@ class Note extends FlxSprite
...
+			switch(CoolUtil.difficultyString()) {
+				case 'Easy':
+					hitHealth = 0.06;
+					missHealth = 0.02;
+				case 'Normal':
+					hitHealth = 0.02;
+					missHealth = 0.04;
+				case 'Hard':
+					hitHealth = 0.01;
+					missHealth = 0.03;
```
Comment (Intancote) [inline source/Note.hx:278]: Don't know if this is intentional or not but is the missHealth supposed to be lower on Hard than Normal?

Comment (Mastercopper2346) [reply source/Note.hx:278]: Yep, it’s intentional. Press 3 notes to make up for that one miss.

Comment (Intancote) [reply source/Note.hx:278]: Alr then

File: source/Note.hx [hunk 2]
```diff
@@ -265,6 +265,18 @@ class Note extends FlxSprite
 				prevNote.updateHitbox();
 				// prevNote.setGraphicSize();
 			}
+			
+			switch(CoolUtil.difficultyString()) {
```
Comment (ImaginationSuperHero528) [inline source/Note.hx:269]: Uh... what about custom difficulties?

Comment (Mastercopper2346) [reply source/Note.hx:269]: Wait

**Comment by Itz-Miles** [comment]
You should still put this in options or gameplay changers. This PR changes gameplay not just for the engine, but every mod you'll ever play on psych, making some harder mods impossible to beat (also mentioning that this will just break most mod compatibility in general)

File: source/Note.hx (hunk 2 above)
Comment (Mastercopper2346) [reply source/Note.hx:269]: k i fixed it :))))

**Comment by Mastercopper2346** [comment]
fine.. i'll work harder on it.

**Comment by frantastic24** [comment]
This is pretty useless considering you can just use gameplay modifiers and/or setProperty

**Pull request closed without merging**